src/config/section/compute.js

```diff
@@ -86,7 +86,7 @@ export default {
           dataView: true,
           groupAction: true,
           show: (record) => { return ['Stopped'].includes(record.state) },
-          args: ['podid', 'clusterid', 'hostid'],
+          args: ['podid', 'clusterid', 'hostid', 'bootintobios'],
```

Comment:
@DaanHoogland is this feature allowed for users as well?

Reply:
yes as i understand it it is